### ISSUE TITLE
Environment variable substitution stopped working in 1.5 and VS Code 1.51

### ISSUE THREAD

**Issue by hriekehof** [microsoft/vscode-cmake-tools#1526]
### Brief Issue Summary

So far i defined a custom build directory via `"cmake.buildDirectory": ${env:MY_BUILD_FOLDER}/${buildKit}/${buildType}`
Today i updated VS Code and the extension and suddenly i get an error from rollbar:

```
[rollbar] Ausnahmefehler: Fehler beim Schreiben in Datei "${env:MY_BUILD_DIR}\xxx\RelWithDebInfo\.cmake\api\v1\query\client-vscode\query.json". Error: ENOENT: no such file or directory
```

It seems that the variable substitution for environment variables is not working anymore. When i use just `${workspaceFolder}` for example everything works. 

### Platform and Versions

- **Operating System**: Windows 
- **CMake Version**: 3.18 
- **VSCode Version**: 1.51 
- **CMake Tools Extension Version**: 1.5 
- **Compiler/Toolchain**: arm-none-eabi-gcc

**Comment by bobbrow** [comment]
Thank you for reporting this.  We will investigate and get a patch out soon.

**Comment by BarrettLowe** [comment]
Maybe related: after updating I cannot resolve cmake variables in the cmake.buildDirectory setting.

${variant:buildType} seems to resolve to nothing...

Does this sound like tit's related? I will open a new issue if not.

**Comment by bobbrow** [comment]
The variant substitution issue is covered by #1525.  I suspect they could be related, but in any case we are looking at both of them.

In the meantime, if you are blocked you can revert back to 1.4.2 by clicking the gear icon next to CMake Tools in the extensions panel and selecting "Install Another Version...":
![image](https://user-images.githubusercontent.com/12818240/98391536-a2068f00-200b-11eb-9d53-290c80602059.png)

**Comment by andreeis** [comment]
This bug fix is included in CMake Tools 1.5.1 which was just released today.
Let us know if you encounter any other problems.

**Comment by hriekehof** [comment]
works perfectly again. thanks for the quick fix 👍
